src/Raven.Server/Smuggler/Documents/StreamDestination.cs

```diff
@@ -417,20 +455,18 @@ public void WriteDocument(DocumentItem item, SmugglerProgressBase.CountsWithLast
                 if (item.Attachments != null)
                     throw new NotSupportedException();
 
-                if (item.Counters != null)
-                    throw new NotSupportedException();
-
                 var document = item.Document;
                 using (document.Data)
                 {
                     WriteUniqueAttachmentStreams(document, progress);
-                    WriteUniqueCounters(document, progress);
 
                     if (First == false)
                         Writer.WriteComma();
                     First = false;
 
-                    document.EnsureMetadata();
+                    //remove HasCounter flag if exists, since its not recognizable in v4.0
+                    document.EnsureMetadata(removeCountersFlag: true);
```

Comment:
We don't want that. 4.0 should be able to just ignore stuff that it doesn't know